src/main/java/com/flab/shoeauction/exception/file/IllegalMimeTypeException.java

```diff
@@ -0,0 +1,5 @@
+package com.flab.shoeauction.exception.file;
+
+public class IllegalMimeTypeException extends IllegalArgumentException{
```

Comment:
커밋전에는 컨벤션 체크를 해주세요. pre commit hook을 쓰셔도 좋습니다~

```suggestion
public class IllegalMimeTypeException extends IllegalArgumentException{
```

Reply:
checkstyle pre commit hook을 적용하려고 하는 중인데 쉘 스크립트 짜는 것이 익지 않아서 시간이 조금 걸리는 것 같습니다..
적용 전까지는 커밋할 때 스타일 체크하도록 노력하겠습니다.